java/Oleksii-Riabov/src/com/zoolatech/lecture2/tasks/_1/Calculator.java

```diff
@@ -0,0 +1,134 @@
+package com.zoolatech.lecture2.tasks._1;
+
+import java.util.Scanner;
+
+/**
+ * Define a class that represents a calculator. The class should provide methods
+ * that accept another value and perform addition, subtraction, multiplication and
+ * division operations on a value stored in a calculator instance. Divides by a
+ * zero should be forbidden and ignored. The class should also provide a method to
+ * get a current value. The class should work with both integer and double
+ * numbers (ignore roundoff errors). Assume all operation results fit into the
+ * range of values for a current value type.
+ */
+
+public class Calculator {
+    double value;
```

Comment:
1. The idea was that you will apply math operations and passed values to the variable `value`.
2. The instance field can be private

Reply:
1. Update task and make "add", "subtract", "multiply", "divide" methods to accept a single value and combine it with a stored value.
2. Make variable "value" private static.